src/main/follower.ts

```diff
@@ -208,6 +207,7 @@ export async function followerId() {
     );
     return identity.id;
   } catch (_error) {
+    logger.log(`followerId: `, _error);
     return '';
```

Comment:
```suggestion
  } catch (error) {
    logger.error(error);
    return '';
```

Guess these are just debugging things for draft anyway

Reply:
exactly, removed